### PR TITLE
Fix systemctl enable neo4j.service documentation

### PR DESCRIPTION
**Pull request by Tavark**
The documentation for neo4j.service systemctl was invalid. 
The command "systemctl daemon-reload" must not be part of the neo4j.service definition.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-903.surge.sh

**Comment by Tavark** [comment]
Looks good to me :)

File: modules/ROOT/pages/installation/linux/tarball.adoc [hunk 1]
```diff
@@ -87,8 +87,13 @@ TimeoutSec=120
...
+----
+
+. Reload systemctl to pick up the new service file:
++
+[source, shell]
+----
+systemctl enable neo4j
 ----
 
 . Configure Neo4j to start at boot time:
```
Comment (jennyowen) [inline modules/ROOT/pages/installation/linux/tarball.adoc:99]: It now tells users to run `systemctl enable neo4j` twice in a row, which looks a bit weird.
Could the line `. Reload systemctl to pick up the new service file:` be merged with the one that says `. Configure Neo4j to start at boot time:` to just be:

`. Reload systemctl to pick up the new service file and configure Neo4j to start at boot time:`

then we just need to tell people to run `systemctl enable neo4j` once.

**Review by jennyowen — CHANGES_REQUESTED** [review]
That's a good catch, thank you for submitting a fix!
I only have one minor change request.

@renetapopova I don't know if there's any license agreement requirements for community users contributing to the documentation?

File: modules/ROOT/pages/installation/linux/tarball.adoc (hunk 1 above)
Comment (Tavark) [reply modules/ROOT/pages/installation/linux/tarball.adoc]: Changed it accordingly.

**Comment by Tavark** [comment]
> That's a good catch, thank you for submitting a fix! I only have one minor change request.
> 
> @renetapopova I don't know if there's any license agreement requirements for community users contributing to the documentation?

Regarding the question: My company has just licensed Neo4j Enterprise.

**Review by jennyowen — APPROVED** [review]
Looks great!

**Comment by renetapopova** [comment]
> > That's a good catch, thank you for submitting a fix! I only have one minor change request.
> > @renetapopova I don't know if there's any license agreement requirements for community users contributing to the documentation?
> 
> Regarding the question: My company has just licensed Neo4j Enterprise.

Thank you both! I'll take it from here.

**Pull request merged**